deck-search/twda.js

```diff
@@ -1,3 +1,44 @@
+class MultiCardSearch {
+    constructor() {
+        this.cardList = new Set()
+    }
+    add(card) {
+        this.cardList.add(card)
+        this.updateView()
+    }
+    remove(card) {
+        this.cardList.delete(card)
+        this.updateView()
+    }
+    clear() {
+        this.cardList.clear()
+        this.updateView()
+    }
+    list() {
+        return Array.from(this.cardList)
+    }
+    updateView() {
+        let cardListNode = document.getElementById('multi-card-search')
+        while (cardListNode.firstChild) {
+          cardListNode.removeChild(cardListNode.firstChild)
+        }
+        for (let card of this.cardList.values()) {
+            let cardNode = document.createElement('li')
+            cardNode.textContent = `${card}`
+            cardNode.addEventListener("click", function (e) {
+                removeMultiCardSearchCard(card)
+            })
+            cardListNode.append(cardNode)
+        }
+    }
```

Comment:
Nothing that's really mandated here, but maybe including the search params in the URL (and window history), could be a cool feature. Maybe that can come next though 😋

Reply:
We can register a new issue for this.